src/controllers/user.js

```diff
@@ -98,3 +97,20 @@ export const updateUserCohortById = async (req, res) => {
     return sendMessageResponse(res, 500, 'Unable to update user')
   }
 }
+
+export const getStudentWithoutCohort = async (req, res) => {
+  try {
+    const userWithoutId = await dbClient.user.findMany({
```

Comment:
Rather than doing the prisma query directly here, I think it would be worth doing it in `domain/user.js` so that it's consistent with the rest of the code. For example, we have a "findManyByFirstName" on doman/user - we could add a "findManyStudentsWithoutCohort" or similar. This will need a few more changes in the domain/user - we can go through them togehter on a call this morning if you like?

Reply:
Done ✅